File: Nível 2/1074 - Par ou Ímpar/main.lua
quantia = tonumber(io.read())

for i = 1, quantia do 
    valor = tonumber(io.read())

    if valor == 0 then
        print("NULL")
    elseif valor % 2 == 0 and valor < 0 then
        print("EVEN NEGATIVE")
    elseif valor % 2 == 1 and valor < 0 then
        print("ODD NEGATIVE")
    elseif valor % 2 == 0 and valor > 0 then
        print("EVEN POSITIVE")
    elseif valor % 2 == 1 and valor > 0 then
        print("ODD POSITIVE")
    end
end

File: Nível 2/1074 - Par ou Ímpar/main.py
quantia = int(input("A: "))

for i in range(1, quantia + 1):
    valor = int(input("B: "))

    if valor == 0:
        print("NULL")
    elif valor % 2 == 0 and valor < 0:
        print("EVEN NEGATIVE")
    elif valor % 2 == 1 and valor < 0:
        print("ODD NEGATIVE")
    elif valor % 2 == 0 and valor > 0:
        print("EVEN POSITIVE")
    elif valor % 2 == 1 and valor > 0:
        print("ODD POSITIVE")